### PR TITLE
add event from inspector docs

### PR DESCRIPTION
**Pull request by bjornj12**
Docs on how adding events and properties from the inspector works

### CODE REVIEW

**Comment by hugihlynsson** [comment]
#### 📝 Changed routes:


| [**…-events-from-inspector**](https://docs-bsbdftu4k.avo-dev.app/workspace/add-events-from-inspector) | [**/workspace/inspector**](https://docs-bsbdftu4k.avo-dev.app/workspace/inspector) |
|:-:|:-:|
|<a href="https://docs-bsbdftu4k.avo-dev.app/workspace/add-events-from-inspector">Screenshot of /workspace/add-events-from-inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/9xY-iU8vhzdrtZi-Ot63eY8AyBuyxZCl-r6xiSwbkJWssMVzexrzyur9WuCE8l1y1BncDchuyN3B5Cv0XrPSjg.png">(view full size)</a> |<a href="https://docs-bsbdftu4k.avo-dev.app/workspace/inspector">Screenshot of /workspace/inspector</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/B8yfyoQkGlS29z5tWjyHnnaV3qCBLKKwrtm998YUG498tNvl_67fWEdrTToUfcVltQMNVzSEIjLjU4OcTL_9ZA.png">(view full size)</a> |







Commit 200d964b4de8ccde2bd3da449d5bcec8742bffbe (https://docs-bsbdftu4k.avo-dev.app).

File: pages/workspace/add-events-from-inspector.mdx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    alt="Image shows a confirm dialog to make sure the user wants to add this event"
+  />
+</center>
+
+Adding the event to the tracking plan will create the Event and do the following:
+
+- Add all sources the event has according to the inspector
+- Add properties that exist in the tracking plan and are on the event in the inspector.
+- Create and add properties that are missing from the tracking plan but are on the event in the inspector
+  - If the property already exists in different casing, it will not be created.
```
Comment (alex-tpom6oh) [inline pages/workspace/add-events-from-inspector.mdx:52]:
```suggestion
  - If the property already exists in different casing, the existing property will be added to the event.
```

File: pages/workspace/add-events-from-inspector.mdx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  - If the property has reported seeing 2 types (int, string for example) it will select the type that Inspector has seen more of.
+  - Each property will be created as required
```
Comment (alex-tpom6oh) [inline pages/workspace/add-events-from-inspector.mdx:54]: What exactly does this mean?

File: pages/workspace/add-events-from-inspector.mdx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  - All new properties created will be created as **Event properties**
+
+### Add Property
+
+Adding a Property directly from the inspector will create the Property and do the following:
+
+- Validate that the property does not exist already in the workspace
+  - If a property with different casing already exists, a window will pop up asking you to confirm your creation of the property.
+- If the property has reported seeing 2 types (int, string for example) it will select the type that Inspector has seen more of.
+- Each property will be created as required
```
Comment (alex-tpom6oh) [inline pages/workspace/add-events-from-inspector.mdx:64]: Samer here?

File: pages/workspace/add-events-from-inspector.mdx (hunk 1 above)
Comment (bjornj12) [reply pages/workspace/add-events-from-inspector.mdx:52]: It wont be added, we will suggest not creating it with a confirm modal, as this indicates an error in the implementation

File: pages/workspace/add-events-from-inspector.mdx (hunk 2 above)
Comment (bjornj12) [reply pages/workspace/add-events-from-inspector.mdx]: fixed it to `created with presence "Always Sent"`

File: pages/workspace/add-events-from-inspector.mdx (hunk 1 above)
Comment (alex-tpom6oh) [reply pages/workspace/add-events-from-inspector.mdx:52]: Ok, got it. and what would be the result? Would the property be ignored in the tracking plan?

File: pages/workspace/add-events-from-inspector.mdx (hunk 2 above)
Comment (alex-tpom6oh) [reply pages/workspace/add-events-from-inspector.mdx]: Ah right

File: pages/workspace/add-events-from-inspector.mdx (hunk 1 above)
Comment (bjornj12) [reply pages/workspace/add-events-from-inspector.mdx:52]: It's just not created, making it red in the dashboard with an "add" button which will trigger a confirm dialog on press, showing them a similar property exists

**Review by alex-tpom6oh — APPROVED** [review]
Great docs!

**Pull request merged**